apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -15,6 +16,7 @@ export class GetNotificationsFeed {
     private subscriberRepository: SubscriberRepository
   ) {}
 
+  @Cached('messages-feed')
```

Comment:
Addressing it on this PR and not on the previous PRs because I wanted to present the example.
At the current state, the decorator can work with an empty string as a `key`, and the result will be the usecase class name (if it is usecase otherwise the function name), however, I want to make the input mandatory.
the reasons for it are: 
1. first of all that way it is clear the cache key prefix the decorator will use.
2. it will be easier for the user of the invalidation decorator to understand what `key` to add.
3. if the class/function name will change the `key` won't change.
LMKWYT

Reply:
In addition, the key here is still WIP, still not sure what the exact prefix it should be.